docs/source/install.ipynb

```diff
@@ -16,6 +16,8 @@
     "pip install evalml\n",
     "```\n",
     "\n",
+    "For Python version 3.6, ensure that versions for `pip >= 18.0` and `setuptools > 40.0.0` in order to install evalml\n",
```

Comment:
Maybe: "For Python versions 3.6.X you must install `pip >= 18.0` and `setuptools > 40.0.0` in order to install evalml"